uplink/config.go

```diff
@@ -150,3 +156,48 @@ func (c Config) GetEncryptionScheme() storj.EncryptionScheme {
 		BlockSize: int32(c.Enc.BlockSize),
 	}
 }
+
+// LoadEncryptionKey loads the encryption key stored in the file pointed by
+// filepath.
+//
+// An error is returned if filepath is empty, file is not found or there is an
+// I/O error.
+func LoadEncryptionKey(filepath string) (key *storj.Key, error error) {
+	if filepath == "" {
+		return nil, errors.New("filepath cannot be empty")
+	}
+
+	file, err := os.Open(filepath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("not found key file %q", filepath)
+		}
+
+		return nil, err
+	}
+
+	defer func() { err = errs.Combine(err, file.Close()) }()
+
+	// TODO: if/v3-1541#3 should we check if the file contains a values of length
+	// storj.KeySize??
```

Comment:
Let me know if you think that we should add this check, if it yes, I will add it, otherwise I will just remove the this `TODO` comment.